tests/drop_counters/test_drop_counters.py

```diff
@@ -161,6 +161,34 @@ def enable_counters(duthost):
                 duthost.command("counterpoll {} disable".format(port))
 
 
+@pytest.fixture
+def mtu_setup(duthost):
+    """ Fixture which prepare port MTU configuration for 'test_ip_pkt_with_exceeded_mtu' test case """
+    class MTUConfig(object):
+        iface = None
+        @classmethod
+        def set_mtu(cls, mtu, iface):
+            if "PortChannel" in iface:
+                res = duthost.command("redis-cli -n 4 hset \"PORTCHANNEL|{}\" mtu {}".format(iface, mtu))["stdout"]
+            elif "Ethernet" in iface:
+                res = duthost.command("redis-cli -n 4 hset \"PORT|{}\" mtu {}".format(iface, mtu))["stdout"]
+            if res != "0":
```

Comment:
what does it mean? This is "redis hset", how it is possible to fail here?

Reply:
Who knows, it is executed via ansible fixture which uses ssh, so here is verified return code.

Reply:
This is not an exit code of process, since you get it from stdout.
hset outputs 0, when no new keys were created
hset outputs 1, when one new key was created, so this is not an error

Reply:
If something will fail based in above comment, "0" will not be in stdout.
Anyway this will be fixed as you suggest.

Reply:
Fixed.